### PR TITLE
Added tentative schedule

### PR DESCRIPTION
**Pull request by AnXh3L0**

### CODE REVIEW

File: summit/2018.html [hunk 1]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
         <a href="http://oficina.al" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/oficina.svg" width="250px" height="auto" alt="Ofiçina"/></a><br />
         <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank" rel="noopener">Rruga Asim Vokshi<br />Tirana, Albania</p><img class="map" src="/images/summit/map.jpg" width="600px" height="347px" alt="Map"/></a>
       </div>
+
+      <div class="row" style="margin-bottom: 30px; margin-top: 30px;">
```
Comment (evalica) [inline summit/2018.html]: Not ideal to have inline styling

File: summit/2018.html [hunk 2]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
...
+      <div class="col-md-10 col-md-offset-1">
+        <img class="sponsor" src="/images/summit/agenda.svg" width="65px" height="auto"
```
Comment (evalica) [inline summit/2018.html]: not really a sponsor, broken semantics

File: summit/2018.html [hunk 3]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
...
+            alt="Sponsor"/>
+        <h2 class="text">Schedule</h2>
+        <p class="data"><strong>2 November</strong><br/>
+        4 PM - 8 PM: Reception and Planning for the Weekend Topics</p>
+        <p class="data"><strong>3 November</strong><br/>
+        10 AM-  1 PM : Open Source Design Meta Discussions<br />
```
Comment (evalica) [inline summit/2018.html]: space missing after "AM"

Comment (evalica) [reply summit/2018.html]: + extra space after PM :)

Comment (evalica) [reply summit/2018.html]: you could have used individual <p> instead of using <br/>

File: summit/2018.html [hunk 4]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
...
+        1 PM - 2 PM: Lunch<br />
+        2 PM - 8 PM: Unconference Sessions</p>
+        <p class="data"><strong>4 November</strong><br />
+        10 AM - 1 PM: Working Sessions HandsOn<br />
+        1 PM - 2 PM Lunch<br />
```
Comment (evalica) [inline summit/2018.html]: Missing ":", same on line 54

File: summit/2018.html [hunk 5]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
...
+            alt="Sponsor"/>
+        <h2 class="text">Schedule</h2>
+        <p class="data"><strong>2 November</strong><br/>
+        4 PM - 8 PM: Reception and Planning for the Weekend Topics</p>
+        <p class="data"><strong>3 November</strong><br/>
+        10 AM-  1 PM : Open Source Design Meta Discussions<br />
+        1 PM - 2 PM: Lunch<br />
+        2 PM - 8 PM: Unconference Sessions</p>
+        <p class="data"><strong>4 November</strong><br />
+        10 AM - 1 PM: Working Sessions HandsOn<br />
```
Comment (evalica) [inline summit/2018.html]: HandsOn = Hands-on
Weekend (American) = Week-end (GB) but works :) I'm just being picky

File: summit/2018.html [hunk 6]
```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
...
+        1 PM - 2 PM Lunch<br />
+        2 PM - 6 PM Working Sessions & Open Source Design in 2019</p>
+        </div>
```
Comment (evalica) [inline summit/2018.html]: Would have liked a left-align for the agenda (in order to easily scan the time), but the whole page is centered, so it's ok.

**Comment by evalica** [comment]
@AnXh3L0 you know you rock :)

File: summit/2018.html (hunk 1 above)
Comment (AnXh3L0) [reply summit/2018.html]: Fixed

File: summit/2018.html (hunk 2 above)
Comment (AnXh3L0) [reply summit/2018.html]: Changed

File: summit/2018.html (hunk 3 above)
Comment (AnXh3L0) [reply summit/2018.html]: Fixed spacing

File: summit/2018.html (hunk 4 above)
Comment (AnXh3L0) [reply summit/2018.html]: Added the :

File: summit/2018.html (hunk 5 above)
Comment (AnXh3L0) [reply summit/2018.html]: Also fixed this

File: summit/2018.html (hunk 6 above)
Comment (AnXh3L0) [reply summit/2018.html]: Yeah, it's a problem because the whole page is centered

**Comment by elioqoshi** [comment]
Thanks @AnXh3L0 and @evalica for the quick review. We can always tweak but good to have something up now

**Comment by elioqoshi** [comment]
@AnXh3L0 just checking in live now. The schedule should be left aligned (doesn't matter if everything else is middle aligned). https://ux.stackexchange.com/questions/119684/why-center-align-text-in-2018

It's aesthetics against accessibility, so I'd prioritize latter.

Can you also make the times bold?

**Pull request merged**